build.gradle

```diff
@@ -20,11 +20,11 @@ dependencies {
   compile 'io.dropwizard.modules:dropwizard-flyway:1.3.0-4'
   compile 'io.dropwizard:dropwizard-core:1.3.7'
   compile 'io.dropwizard:dropwizard-jdbi3:1.3.7'
+  compile 'io.dropwizard.metrics:metrics-jdbi3:4.1.0-rc3'
   compile 'org.jdbi:jdbi3-postgres:3.5.1'
   compile 'org.jdbi:jdbi3-sqlobject:3.5.1'
   compile 'org.postgresql:postgresql:42.2.5'
   compileOnly 'org.projectlombok:lombok:1.18.4'
-
```

Comment:
Very minor: The newline here acts as a separator and helps distinguish between application and test dependencies. Mind adding it back?